### ISSUE TITLE
Dialog accessibility 

### ISSUE THREAD

**Issue by mnelson4** [mnelson4/printmyblog#276]
The design details and add new print 
materials dialogs should be accessible:
-have the dialog aria role
-receive focus when opened
-return focus when closed

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role

while im there, I should deselect items when add new print materials dialog is opened
